### DISCUSSION TITLE
importing simbrief flight plan

### DISCUSSION THREAD

**Discussion by beep747** [flybywiresim/aircraft#3107] [Q&A]
I watched every tutorial on this subject, I tried every step I could think of, but it just doesn't work. 
The fuel and payload are taken from Simbrief, but when choosing INIT, INIT REQUEST doesn't show. 
So, my departure and destination, Fuel and Payload are taken from Simbrief but no flightplan.

Any help would be much appreciated.
Best regards,
Jos

**Comment by ChruutvoLuzi** [comment]
Are you running a recent version? And if yes, does the Simbrief Username Field show up in the AOC Page?

**Comment by beep747** [comment]
Hello,
I always use the latest Developers version. 
I open SIMBRIEF in my browser, enter the Flight info and AIrcraft type and push Generate OFP.
Then, I run MSFS 2020. On the welcome screen, I enter the departure and arrival airport, choose a gate and start the flight.
Batteries and 
![mcdu](https://user-images.githubusercontent.com/70146852/105047406-92271480-5a6a-11eb-947e-84c8b2b9b434.JPG)
ground power set to on.
I follow the tutorial. I get the departure and arrival airport, the amount of fuel and the payload according to my SIMBRIEF plan but when choosing INIT, INIT request is not an option.
My SiMBRIEF username is set correctly and shows up.
I must doing something wrong but I don't have a clue what.
Best regards,
Jos

**Reply by ChruutvoLuzi** [reply]
try it without entering the flightplan in the world screen and only choosing a gate as departure, all fields on INIT A have to be blank in order to show INIT REQUEST

**Comment by beep747** [comment]
That was the problem! Thank you so much for your help.
I didn't realize that I generated a flightplan by completing the departure and arrival airport.
Best regards,
Jos

**Reply by ChruutvoLuzi** [reply]
Glad i could help, btw, i think you can select an answer that solved the issue somewhere in here. Would help keeping things clean on Github :D
